jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Checkbox.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.jdi.light.material.elements.inputs;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.material.asserts.inputs.CheckboxAssert;
+
+public class Checkbox extends UIBaseElement<CheckboxAssert> {
+
+    @JDIAction("Click on '{name}'")
+    public void clickOnCheckbox() {
+        core().click();
+    }
+
+    @JDIAction("Is '{name}' selected")
+    public boolean isSelectedCheckbox() {
+        return core().hasClass("Mui-checked");
+    }
+
+    @JDIAction("Is '{name}' not selected")
+    public boolean isNotSelectedCheckbox() {
+        return !isSelectedCheckbox();
+    }
+
+    @JDIAction("Is '{name}' selected")
+    public boolean isEnabledCheckbox() {
+        return core().isEnabled();
+    }
+
+    @Override
+    public CheckboxAssert is() {
+        return new CheckboxAssert().set(this);
+    }
+
+
```

Comment:
please add methods to check and uncheck checkbox in addition to click
smth like
```
public void check() {
if (isNotSelectedCheckbox()) {
click()
}
```